platforms-whitepaper/v1alpha1/paper.md

```diff
@@ -274,6 +263,14 @@ following which implementers should keep in mind.
 1. Platform teams must seek support of enterprise leadership and show impact on
    value streams
 
+When adopting platforms, choosing the right capabilities and experiences to
+enable first, can be crucial. Capabilities that are frequently required and
+undifferentiated, like pipelines, databases and observability, may be a good
+place to start. Platform teams may also choose to focus first on a limited number
+of engaged and skillful app teams. Detailed feedback from such teams improves the
+first platform experiences; and people from those teams help champion and
+evangelize the platform to later adopters.
+
 Perhaps most important is to recognize that the success of a platform is
```

Comment:
WDYT of expanding the first sentence here along these lines:

"Perhaps most important is to _treat the platform as a customer-facing **product** and recognize that its success_ is directly dependent on..."

Does this better connect this paragraph to the concept of "platform as a product"?

Reply:
```suggestion
Perhaps most important is to treat the platform as a customer-facing product and recognize that its success  is
```